tests/.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
From what I can tell, these are smoke tests that run through the core functionality, 1 for items and 1 for staking. I think we could just keep our original directory structure and put them side by side: https://github.com/raindrops-protocol/raindrops/tree/master/rust/tests - this way we have tests for items, staking and namespaces which can all be run via `anchor test`. And the nice utils you created we can reuse them in all of our tests.

The current directory setup makes it so now we have 2 places where we have to keep track of test code for our programs and IMO would be better kept in 1 place.

Reply:
In `rust/tests`, we can write program unit tests. And we can put system-wide tests in root/`tests` directory since system-wide tests can include other tests, not just program tests.
And `anchor test` can only be used in program tests.

Reply:
Actually, `anchor test` does two things:
- run `anchor build`
- run `yarn test`

In our system-wide tests, we should run local validator manually to load several programs (token_metadata program, raindrops programs) and also should run yarn build (build programs, build sdk).

So we only need to run `yarn test`.

Reply:
Anchor test already loads the programs we need, including the metaplex ones, i set that up for us: https://github.com/raindrops-protocol/raindrops/blob/master/rust/Anchor.toml#L38

Reply:
Not to nitpick wording but since you are splitting these tests into separate files depending on the program, you have Item, Staking, Matches they feel like more integration tests on a per program basis than full system tests. I actually think this is good, we should build up to system tests by first testing each program works how its designed. IMO we should move these to be side-by-side to the namespace tests, then we have integration tests for all components and can then tackle system tests.

Reply:
I do not want us to have to bring up any local validator manually. Anchor does everything we need for the validator environment for these tests.